src/core/connection.c

```diff
@@ -5781,7 +6005,7 @@ QuicConnParamGet(
         }
 
         *BufferLength = sizeof(QUIC_SETTINGS);
-        *(QUIC_SETTINGS*)Buffer = Connection->Settings;
+        CxPlatCopyMemory(Buffer, &Connection->Settings, *BufferLength); // TODO: How to copy out DesiredVersionsList?
```

Comment:
Fix this. At least remove comment.

Reply:
How about informing the app they need to provide a buffer large enough to store the `DesiredVersionsList` as well as the `QUIC_SETTINGS`?